--- a/Makefile.am
+++ b/Makefile.am
@@ -157,7 +157,7 @@ src/plugin.$(OBJEXT): src/builtin.h
 src/builtin.h: src/genbuiltin $(builtin_sources)
 	$(AM_V_GEN)$(srcdir)/src/genbuiltin $(builtin_modules) > $@
 
-$(src_neard_OBJECTS) $(tools_nfctool_nfctool_OBJECTS) $(plugin_objects): $(local_headers)
+$(src_neard_OBJECTS) $(tools_nfctool_nfctool_OBJECTS) $(plugin_objects) $(tools_snep_send_OBJECTS) $(unit_test_ndef_parse_OBJECTS) $(unit_test_snep_read_OBJECTS) $(unit_test_ndef_build_OBJECTS): $(local_headers)
 
 include/near/version.h: include/version.h
 	$(AM_V_at)$(MKDIR_P) include/near
